packages/augur-simplified/src/modules/App.tsx

```diff
@@ -25,9 +27,13 @@ const AppBody = () => {
   const {
     sidebarType,
     showTradingForm,
-    actions: { setIsMobile, updateGraphData, updateProcessed },
+    processed,
+    loginAccount,
+    actions: { setIsMobile, updateGraphData, updateProcessed, updateUserBalances },
   } = useAppStatusStore();
 
+  const { library } = useActiveWeb3React()
```

Comment:
You don't need to use `useActiveWeb3React()`, all that is stored inside the main app context
```
const { chainId, library, active } = loginAccount;
```

Reply:
removed refer to useActiveWeb3React in app.tsx